src/person/person.model.ts

```diff
@@ -1,51 +1,51 @@
 import * as mongoose from 'mongoose';
-import { IUser } from './user.interface';
-import { UserValidate } from './user.validate';
+import { IPerson } from './person.interface';
+import { PersonValidate } from './person.validate';
 import { RESPONSIBILITY } from '../utils';
 
 (<any>mongoose).Promise = Promise;
 const ObjectId = mongoose.Schema.Types.ObjectId;
 
-const userValidator = new UserValidate();
+const personValidator = new PersonValidate();
 
 function validateEmail(email: string): boolean {
   return true;
 }
 
-export const UserSchema = new mongoose.Schema(
+export const PersonSchema = new mongoose.Schema(
   {
     identityCard: {
       type: String,
       required: [true, 'You must enter an identity card!'],
       unique: true,
-      validate: { validator: UserValidate.identityCard, message: '{VALUE} is an invalid identity card!' },
+      validate: { validator: PersonValidate.identityCard, message: '{VALUE} is an invalid identity card!' },
     },
     personalNumber: {
       type: String,
       unique: true,
       sparse: true,
-      validate: { validator: UserValidate.personalNumber, message: '{VALUE} is an invalid personal number!' },
+      validate: { validator: PersonValidate.personalNumber, message: '{VALUE} is an invalid personal number!' },
     },
-    primaryUser: {
+    primaryPerson: {
       type: String,
-      required: [true, 'You must enter a primary user!'],
+      required: [true, 'You must enter a primary person!'],
       unique: true,
-      validate: { validator: UserValidate.email, message: '{VALUE} is an invalid User' },
+      validate: { validator: PersonValidate.email, message: '{VALUE} is an invalid Person' },
     },
-    secondaryUsers: [{
+    secondaryPersons: [{
```

Comment:
Should be "domainUsers" or "secondaryDomainUsers"

Reply:
Done